### PR TITLE
MB-15752 add and implement BuildPPMShipment

### PR DESCRIPTION
**Pull request by mlm55**
## Summary

Add and implement BuildPPMShipment

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

```sh
make server_test e2e_test
```

### CODE REVIEW

File: pkg/factory/ppm_shipment_factory.go [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+)
+
+type ppmBuildType byte
+
+const (
+	ppmBuildStandard ppmBuildType = iota
+	ppmBuildMinimal
+)
+
+// buildServiceMemberWithBuildType does the actual work
```
Comment (ahobson) [inline pkg/factory/ppm_shipment_factory.go:20]: Maybe some copy pasta here with `buildServiceMember` reference?

File: pkg/factory/duty_location_factory.go [hunk 2]
```diff
@@ -176,6 +176,14 @@ func FetchOrBuildOrdersDutyLocation(db *pop.Connection) models.DutyLocation {
...
 				},
 			},
+			{
+				Model: models.Address{
+					City:       "Augusta",
+					State:      "GA",
+					PostalCode: "30813",
+				},
+				Type: &Addresses.DutyLocationAddress,
+			},
```
Comment (ahobson) [inline pkg/factory/duty_location_factory.go:186]: I'm not sure I'm following why this was added

Comment (mlm55) [reply pkg/factory/duty_location_factory.go:186]: The original FetchOrMakeDefaultNewOrdersDutyLocation used this address for the default new duty location.  MakePPMShipment uses the new orders duty location postal code to set `DestinationPostalCode` and some of the tests relied on `30813` being the `DestinationPostalCode`

Comment (ahobson) [reply pkg/factory/duty_location_factory.go:186]: How would you feel about adding a comment briefly explaining that?

**Review by ahobson — APPROVED** [review]
🚀

**Pull request merged**